ui/src/index.html

```diff
@@ -13,6 +13,49 @@
     <script src="https://apis.google.com/js/platform.js"></script>
     <link href="https://fonts.googleapis.com/css?family=Montserrat:400,600,700" rel="stylesheet">
     <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+
+    <script type="text/javascript">
+      // This data configures the Zendesk web widget with settings to show only
+      // the "contact us" form. See https://developer.zendesk.com/embeddables/docs/widget/
+      // for API docs.
+      window.zESettings = {
+        webWidget: {
+          chat: {
+            suppress: true,
+          },
+          color: {
+            // This is an AoU dark purple color.
+            theme: "#262262"
+          },
+          contactForm: {
+            attachments: true,
```

Comment:
One of the things we lose with this approach is the ability to auto-attach notebook logs:
![screen shot 2019-01-18 at 4 17 24 pm](https://user-images.githubusercontent.com/116679/51413361-b3339280-1b3c-11e9-819b-f3254d49c321.png)
Please confirm with Karthik if this is an acceptable trade-off. Not sure if there's a way to accomplish the same feat via this widget.